lib/config.js

```diff
@@ -8,6 +8,14 @@ require('bedrock-validation');
 
 const cfg = config['authn-token-http'] = {};
 
+// client id to be baked into cookies that is used in client registration
+cfg.cid = {
+  defaults: {
+    // 30 days expiration
+    maxAge: 30 * 24 * 60 * 60 * 1000,
+  }
+};
```

Comment:
There might be a need to configure the name of this cookie, so we shouldn't make its name the same thing as the configuration variable. Also, we need a short name for the cookie itself to save overhead on network data, but we don't need that here in the config, so we should be verbose. We might also want some future proofing here, so let's name space the client cookie info under `cookies` as well. Lastly, do we need "defaults"? I've removed that field in my suggestion.

```suggestion
cfg.cookies = {};
cfg.cookies.clientId = {
  name: 'cid',
  // 30 days expiration
  maxAge: 30 * 24 * 60 * 60 * 1000
};
```